modyn/tests/config/test_config_integrity.py

```diff
@@ -0,0 +1,55 @@
+from pathlib import Path
+
+import pytest
+from modyn.config import read_modyn_config, read_pipeline
+from modynclient.config import read_client_config
+
+"""Directories can either be file paths or directories.
+If they are directories, all yaml files in the directory will be used."""
+
+MODYN_PIPELINE_CONFIG_PATHS: list[str] = [
+    "modynclient/config/examples/dummy.yaml",
+    "modynclient/config/examples/mnist.yaml",
+    "benchmark/",
+]
+
+MODYN_SYSTEM_CONFIG_PATHS = ["modyn/config/examples/modyn_config.yaml"]
+
+MODYN_CLIENT_CONFIG_PATHS = [
+    "modynclient/config/examples/modyn_client_config.yaml",
+    "modynclient/config/examples/modyn_client_config_container.yaml",
+]
+
+
+def _discover_files(paths: list[str]) -> list[Path]:
+    files = []
+    for path in paths:
+        p = Path(path)
+        if not p.exists():
+            continue
+        if p.is_dir():
+            files.extend(p.rglob("*.yaml"))
```

Comment:
maybe also yml? I feel like we are not 100% consistent with yaml vs yml as an extension